{"name": "test_fail_password_empty[--对不起！请输入密码！]", "status": "passed", "attachments": [{"name": "stdout", "source": "45247307-d070-43ec-b8c2-5270d788131d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "''"}, {"name": "expect", "value": "'对不起！请输入密码！'"}], "start": 1601430956737, "stop": 1601430965588, "uuid": "7163a471-dc34-4706-9e1b-bd4145e7b487", "historyId": "19c20d6591c707086577ec0f634f58fd", "testCaseId": "e048db9676071fc597e1087551b47302", "fullName": "TestCases.test_1_loginCase.TestLogin#test_fail_password_empty", "labels": [{"name": "tag", "value": "Test1"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_1_loginCase"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "HPTSJ003"}, {"name": "thread", "value": "8700-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_1_loginCase"}]}